ui/src/components/FlightsTable.tsx

```diff
@@ -181,7 +160,7 @@ class FlightsTable extends React.Component<IProps, IState> {
         this.getFlightsData(endpoint, this.updateFlightsData)
     }
 
-   private updateFlightsWithoutPort() {
+    private updateFlightsWithoutPort() {
         let endpoint = this.flightsPortEndPoint(this.props.region, this.props.post, this.props.country, this.props.date, this.props.timezone);
```

Comment:
It would be good practice to switch `let`s to `const`s as it's immutable (analogous to `var` vs `val`)